ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -14077,9 +14077,7 @@ ASTNode analyzeCreateTable(
       storageFormat.setStorageHandler(defaultStorageHandler);
       isUserStorageFormat = true;
     }
-
-    // CREATE TABLE is a DDL and yet it's not handled by DDLSemanticAnalyzerFactory
-    // FIXME: HIVE-28724
+    
     queryProperties.setQueryType(QueryProperties.QueryType.DDL);
```

Comment:
if the this patch works properly, this line is not needed anymore:
```
    queryProperties.setQueryType(QueryProperties.QueryType.DDL);
```

and the correctness will be tested automatically here:
https://github.com/apache/hive/blob/f4ab76bf51859f8a2f4e929b2163bb891d17008f/ql/src/test/org/apache/hadoop/hive/ql/parse/TestSemanticAnalyzer.java#L391-L393